packages/protocol/contracts/layer1/core/impl/Inbox.sol

```diff
@@ -266,14 +271,15 @@ contract Inbox is IInbox, IForcedInclusionStore, EssentialContract {
             // 3. Calculate proposal age and bond instruction
             // ---------------------------------------------------------
             Transition memory transitionAtOffset = commitment.transitions[offset];
-            uint256 proposalAge =
-                block.timestamp - transitionAtOffset.timestamp.max(state.lastFinalizedTimestamp);
+            uint256 proposalAge = block.timestamp - transitionAtOffset.timestamp.max(state.lastFinalizedTimestamp);
```

Comment:
maybe we should use uint256 proposalAge = block.timestamp - transitionAtOffset.timestamp;

then inside calculateBondInstruction, we use it against the (state.lastFinalizedTimestamp+_maxProofSubmissionDelay)